x-pack/plugins/maps/public/classes/sources/mvt_single_layer_vector_source/mvt_single_layer_source_settings.tsx

```diff
@@ -133,7 +133,7 @@ export class MVTSingleLayerSourceSettings extends Component<Props, State> {
           label={i18n.translate(
             'xpack.maps.source.MVTSingleLayerVectorSourceEditor.layerNameMessage',
             {
-              defaultMessage: 'Tile layer',
+              defaultMessage: 'Source layer',
```

Comment:
`Source layer` matches the `source-layer` naming in the TileJson-spec and it's corresponding use in the mapbox-gl spec.